services/wireguard/service/service_unix.go

```diff
@@ -89,6 +91,19 @@ func (manager *Manager) ProvideConfig(sessionConfig json.RawMessage, traversalPa
 		return nil, err
 	}
 
+	config.Consumer.DNS = providerIP(config.Consumer.IPAddress).String()
+	dnsServer := dns.NewServer(
+		net.JoinHostPort(config.Consumer.DNS, "53"),
+		dns.ResolveViaConfigured(),
+	)
+
+	go func() {
```

Comment:
I dont think it's a good idea to not to care if DNS wasn't started

Reply:
`dnsServer.Run()` is a blocking function, it returns nothing until fails. We cannot wait it forever, added logging to be able to track any problem with DNS.

Reply:
Ups, then need to do the same in Openvpn DNS server

Reply:
Changed for OpenVPN too.